nats-api/src/main/proto/com/example/ajaxonboardingproject/cinema_hall.proto

```diff
@@ -2,15 +2,17 @@ syntax = "proto3";
 
 package com.example.ajaxonboardingproject;
 
-message CinemaHall {
+option java_multiple_files = true;
+
+message CinemaHallProto {
```

Comment:
Why renamed? The previous one name was better

Reply:
reverted changes)